examples/planning/python/simple_client.py

```diff
@@ -0,0 +1,116 @@
+import grpc
+import sys
+import yaml
+import os
+import os.path
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+DATA_DIR = os.path.join(os.path.dirname(ROOT_DIR), "data")
+BUILDFILES_DIR = os.path.join(ROOT_DIR, "build")
+if BUILDFILES_DIR not in sys.path:
+    sys.path.append(BUILDFILES_DIR)
+
+from build.types_pb2 import (
+    Constraints,
+    Model,
+    SystemConf,
+    Conf,
+    PositionConstraint,
+    AngleBetweenVectorsConstraint,
+)
+from build.generate_id_pb2 import RegisterPlanContextRequest
+from build.generate_id_pb2_grpc import PlanContextRegistryStub
+from build.builder_pb2 import StartBuildFromConfsRequest
+from build.builder_pb2_grpc import IrisBuilderStub
+
+
+def make_constraints(constraints_name):
+    with open(os.path.join(DATA_DIR, "constraints", f"{constraints_name}.yaml")) as f:
+        data = yaml.safe_load(f)
+    pos_constraints = []
+    angle_constraints = []
+    if "pos_constraints" in data:
+        for constraint in data["pos_constraints"]:
+            pos_constraints.append(
+                PositionConstraint(
+                    frame_A=constraint["frame_A"],
+                    frame_B=constraint["frame_B"],
+                    p_AQ_lower=constraint["position_AQ_lower"],
+                    p_AQ_upper=constraint["position_AQ_upper"],
+                    position_BQ=constraint["position_BQ"],
+                )
+            )
+    if "angle_constraints" in data:
+        for constraint in data["angle_constraints"]:
+            angle_constraints.append(
+                AngleBetweenVectorsConstraint(
+                    frame_A=constraint["frame_A"],
+                    frame_B=constraint["frame_B"],
+                    a_A=constraint["a_A"],
+                    b_B=constraint["b_B"],
+                    angle_lower=constraint["angle_lower"],
+                    angle_upper=constraint["angle_upper"],
+                )
+            )
+    return Constraints(
+        pos_constraints=pos_constraints, angle_constraints=angle_constraints
+    )
+
+
+def raw_dmd(dmd_name):
+    with open(os.path.join(DATA_DIR, "dmd", f"{dmd_name}.dmd.yaml")) as dmd:
+        data = dmd.read()
+    return data
+
+
+def make_models():
+    models = []
+    urdf_dir = os.path.join(DATA_DIR, "urdf")
+    for fname in os.listdir(urdf_dir):
+        if fname.endswith(".urdf"):
+            with open(os.path.join(urdf_dir, fname)) as urdf:
+                models.append(
+                    Model(name=os.path.splitext(fname)[0], urdf_raw=urdf.read())
+                )
+    return models
+
+
+def make_id_request(system_name, package_name, dmd_name, constraints_name):
+    return RegisterPlanContextRequest(
+        system_name=system_name,
+        package_name=package_name,
```

Comment:
why do we need `system_name` and `package_name`?

Reply:
discussed IRL; we should be able to deprecate `package_name` soon (as soon as we store all the geo as a shared resource).

Reply:
even further, this is ok to deprecate because the resource locator in the `dmd` specifies the package (if any) implicitly by the path

Reply:
yeah, that infrastructure doesn't exist yet though. if you took this field out, you would not be able to determine the `PlanContextId`, since we currently save the received files (transmitted over gRPC) to a local directory - there will be more work required to enable this "package discovery" for the planning service.

Reply:
ok -- then I would put a note here so the user knows that `package_name` MUST match the uri in the dmd